examples/standalone/README.md

```diff
@@ -260,6 +259,55 @@ iam.googleapis.com \
 --project ${PROJECT_ID}
 ```
 
+### Outputs by personas
+
+These outputs can be interesting for all user:
+
+- Blueprint type
+- Data ingestion bucket name
+- Data ingestion dataflow bucket name
+- Confidential data project id
+- Data governance project id
+- Data ingestion project id
+- Non-confidential data project id
+- Service account: dataflow controller service account
+
+These outputs can be interesting for data analyst group:
+
+- Data ingestion topic name
+- Pubsub writer service account email
+- Storage writer service account email
+
+These outputs can be interesting for data engineer group:
```

Comment:
this persona is interested in the pipeline and anything associated with it such as things from the harness (build jobs, flex templates) and responsible for plumbing the input to the output.